ACCESS-OM2-GMD-Paper-Figs/Fig4-DrakePassageTransport.ipynb

```diff
@@ -1,80 +1,506 @@
 {
```

Comment:
<p>Nice moving the python comments into Markdown!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/345/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>